### PR TITLE
return the ID every after carrying out a SITE activity

### PR DESCRIPTION
**Pull request by Baalmart**
# Hotfix

**_WHAT DOES THIS PR DO?_**

- [x] This just includes the _id to the returned data after creating a maintenance log.
- [x] Also adds a new optional field (maintenanceType) to the activity Schema ....all to cater for maintenance activities.

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
N/A

**_WHAT IS THE LINK TO THE PR BRANCH_**
N/A

**_HOW DO I TEST OUT THIS PR?_**
`cd AirQo-api/src/device-registry`
`npm install`
`npm run dev-mac` for Linux OR `npm run dev-pc` for Windows

Alternatively, one can use docker :
`docker build -t us.gcr.io/airqo-250220/airqo-location-registry-api .`
`docker run -d -p 6000:3000 us.gcr.io/airqo-250220/airqo-device-registry-api`
Since it runs Redis, you might need to use Docker compose for this, please get in touch with @danielogen  on this approach for more assistance.

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
BASE_URL: http://localhost:3000/api/v1/devices/ts/activity?type=maintain&tenant=airqo
BODY:
```
{
  "deviceName": "aq_501",
  "locationName": "loc_1",
  "date": "2020-07-07T16:09:24.611Z",
  "tags":["blowing", "dusting", "sensor placement", "damn"],
   "description":"preventive"
}
```

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
N/A

**_ARE THERE ANY RELATED PRs?_**
N/A

### CODE REVIEW

File: src/device-registry/utils/doActivityHelpers.js [hunk 1]
```diff
@@ -108,15 +108,15 @@ const doLocationActivity = async (
...
               tenant.toLowerCase(),
               "activity",
               LocationActivitySchema
             ).createLocationActivity(activityBody);
-            log.then((log) => {
+            activityLog.then((activityLog) => {
               return res.status(HTTPStatus.OK).json({
                 message: `${type} successfully carried out`,
-                activityBody,
+                activityLog,
```
Comment (Baalmart) [inline src/device-registry/utils/doActivityHelpers.js:119]: Please note that the response object for the maintenance has been renamed from `activityBody` to `activityLog`.
@ja-odur

Comment (ja-odur) [reply src/device-registry/utils/doActivityHelpers.js:119]: Noted

**Review by ja-odur — CHANGES_REQUESTED** [review]
Everything works as expected although the id naming needs to be reverted to `_id` because its used all over the frontend (more like a convention now).

File: src/device-registry/models/location_activity.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const activitySchema = new Schema(
 activitySchema.methods = {
   toJSON() {
     return {
-      _id: this._id,
+      id: this._id,
```
[the remark this answers is not in the document]
Comment (Baalmart) [reply src/device-registry/models/location_activity.js]: DONE. @ja-odur

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by danielogen — APPROVED** [review]
Looks good.. Thanks @Baalmart

**Pull request merged**